testing/ostest/aio.c

```diff
@@ -219,7 +219,7 @@ static int remove_done(void)
         {
           /* Check if the I/O has completed */
 
-          printf("  list[%d]. result = %d\n", i, aiocbp->aio_result);
+          printf("  list[%d]. result = %ld\n", i, aiocbp->aio_result);
```

Comment:
```suggestion
          printf("  list[%d]. result = %zd\n", i, aiocbp->aio_result);
```
`aio_result` type is `ssize_t`, so I think `z` fits best here.

Reply:
Done.